services/enterprise-gateway/kernels/R_ohdsi_docker/scripts/launch_docker.py

```diff
@@ -24,13 +26,15 @@ def launch_docker_kernel(
     # Can't proceed if no image was specified.
     image_name = os.environ.get("KERNEL_IMAGE", None)
     if image_name is None:
-        sys.exit("ERROR - KERNEL_IMAGE not found in environment - kernel launch terminating!")
+        sys.exit(
+            "ERROR - KERNEL_IMAGE not found in environment - kernel launch terminating!")
 
-    # Container name is composed of KERNEL_USERNAME and KERNEL_ID
-    container_name = kernel_id
+    # KERNEL_USERNAME should be user_id or study_id, but if not set, use kernel_id
+    container_name = os.environ.get("KERNEL_USERNAME", kernel_id)
```

Comment:
when the kernel is started from jupyter notebook, kernel username will be the user id
when the kernel is started from /strategus-results, kernel username will be the study id